### DISCUSSION TITLE
Ivory Coast (225) phone format changed, GitHub should upgrade

### DISCUSSION THREAD

**Discussion by Dahkenangnon** [community/community#51205] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi Github

Hi Everyone

So it's simple as: 

- Ivory Coast (+225) phone number changed since a while and it's impossible to setup 2FA with our number
-  The new format contain (without the country code +225) 10 digits XX XX XX XX XX
- Full internatinal format should be +225 XX XX XX XX XX (space are inserted for readability)

Am I missing something ? Do you have a recommendation or solution ?

PS: This issue should be fixed (if any) asap because Github is requiring us 2FA

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Dahkenangnon ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by Dahkenangnon** [reply]
Ok, waiting so for your reply.

**Reply by Dahkenangnon** [reply]
Hi

still unable to activate the 2FA and you now required the 2FA
